department-of-veterans-affairs/va.gov-cms
# VACMS-11698: Last-touched-by-an-editor field not always reflecting updates

## Description

Closes #11698.

## Testing done
Tested locally. 

## Screenshots
![CleanShot 2022-11-23 at 11 54 58](https://user-images.githubusercontent.com/109987005/203637186-884548ca-6ce1-4d3e-8e14-4a35ef3213d3.png)
![CleanShot 2022-11-23 at 11 54 45](https://user-images.githubusercontent.com/109987005/203637193-78759125-af62-4eab-8f6c-b96eccc5072a.png)


## QA steps

When a piece of content is updated/added the Last Saved by and Editor field and the latest change int he Recent changes block should be the same if saved by a human. Select a piece of content, scroll to the bottom, change moderation to Published, add a revision log message such as "testing" and click save. The same timestamp should be reflected at the top of the Recent Changes block and in the Last Saved by an Editor field. If you click on the edit tab again, both times in the form meta in the top right corner should be the same. 

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
